lib/cdo/poste.rb

```diff
@@ -479,6 +479,8 @@ def self.save_attachments(attachments)
     timestamp = DateTime.now.strftime('%Y%m%d_%H%M_%S%L')
     {}.tap do |saved|
       attachments.each do |name, content|
+        # Prevent saving certificates - they are unecessarily filling storage.
+        next if name.include?('certificate')
```

Comment:
related, possibly easier-to-answer question -- do you have a pointer to an email template or codepath that sets the name containing "certificate"? It would be nice to be able to see why this is the right string to search for.